src/workloads/tpch/denormalized/1/Q10_denormalized_1.yml

```diff
@@ -0,0 +1,50 @@
+SchemaVersion: 2018-07-01
```

Comment:
It looks like we have multiple identicalworkloads now:

- src/workloads/tpch/denormalized/1/Q10.yml
- src/workloads/tpch/denormalized/1/Q10_denormalized_1.yml

Is this necessary?